test/integration/util/util.go

```diff
@@ -33,6 +33,8 @@ import (
 	commonutil "k8s.io/minikube/pkg/util"
 )
 
+const kubectlBinary = "kubectl-v1.6.0-alpha.1"
```

Comment:
Are we planning on modifying this with each new kubectl release?  I am just curious why this wasn't installed as `kubectl`, is there compatibility issues?